homeassistant/components/media_player/__init__.py

```diff
@@ -512,6 +528,22 @@ def shuffle(self):
         """Boolean if shuffle is enabled."""
         return None
 
+    @property
+    def group_candidates(self):
+        """List of players which can be grouped together."""
+        if self.support_grouping:
+            return [
+                player.entity_id
+                for player in self.hass.data[DOMAIN].entities
+                if isinstance(player, self.__class__) and player != self
+            ]
+        return []
+
+    @property
+    def group_members(self):
```

Comment:
If the entity supports grouping, we should make sure that we include this as an attribute in `device_attributes`.

Reply:
Right now `group_members` is always exposed. Do you mean we should only expose it if the device actually supports grouping? Or do you mean we should expose an additional attribute `support_grouping`?

Reply:
Ah I see you added it to `ATTR_TO_PROPERTY`. Let's add it manually to `state_attributes` because we want to only add this attribute if we support grouping.

Reply:
OK, that makes perfectly sense. Fixed.